Makefile

```diff
@@ -203,19 +203,19 @@ $(1)-lint: $(GEN_RS) $(GEN_TS)
 	$$(cargo) clippy  -p $(1) --all-features -- -D clippy::all -D warnings
 
 .PHONY: $(1)-test
-$(1)-test: $(GEN_RS) $(GEN_TS) auraed
```

Comment:
I'm not certain why `auraed` was being built as a dependency in every one of these, so I may be missing something?

Reply:
At first I thought `auraed` was used to run the tests, but it works fine without it.

Reply:
Yeah I'm thinking that was our intent. But we can at least add them back to `X-test-integration` when we actually have integration tests 😄

Reply:
there was at least one point where auraed needed to be running for some tests to run.